---
title: The MicroRNAs-Transcription Factors-mRNA Regulatory Network Plays an Important Role in Resistance to Cold Stress in the Pearl Gentian Grouper
coverWidth: 1920
coverHeight: 1080
date: 2022-01-09
tags: Science
cover: https://benben-miao.gitee.io/image-cloud/BioBlog/paper2-figure9.png
---

<!-- <div style="background-color: #eeeeee; width: 120px; padding:5px 20px; border-radius: 3px;">Read More</div> -->
<!-- more -->

## 
#### Paper Citation
<div class="card">
<b>The MicroRNAs-Transcription Factors-mRNA Regulatory Network Plays an Important Role in Resistance to Cold Stress in the Pearl Gentian Grouper</b>

Miao B-B, Niu S-F, Wu R-X, Liang Z-B and Zhai Y (2022) The MicroRNAs-Transcription Factors-mRNA Regulatory Network Plays an Important Role in Resistance to Cold Stress in the Pearl Gentian Grouper. Front. Mar. Sci. 8:824533. doi: 10.3389/fmars.2021.824533 

https://www.frontiersin.org/articles/10.3389/fmars.2021.824533/full
</div>

## 
#### Abstract
<div class="card">
Pearl gentian grouper (<i>Epinephelus fuscoguttatus</i> ♀ × <i>E.lanceolatus</i> ♂) is a hybrid fish with high commercial value. It is widely cultured on the Asian coast; however, it is not cold-tolerant. Although we have previously characterized the liver transcriptomic responses of this grouper to cold stress, the roles of miRNAs and transcription factors (TFs) in cold resistance and the underlying regulatory mechanisms are still unclear. In this study, we integrated miRNA and mRNA sequencing data for pearl gentian grouper under cold stress and constructed a miRNA-TF-mRNA regulatory network. Furthermore, we screened seven key miRNAs (i.e., gmo-miR-221-5p, ssa-miR-7132b-5p, ola-let-7c, ssa-miR-25-3-5p, ccr-miR-489, gmo-miR-10545-5p, ccr-miR-122) that regulated target genes (including TF ACSS2, TF PPARD, TF PPP4CB; CYP2J2, EHHADH, RXRs, NR1D2, PPP1CC-A, PPP2R1A, FOXK2, etc.). These miRNAs participated in several important pathways and biological processes by the direct or indirect regulation of target genes, such as antioxidation and membrane fluidity, glucose and lipid metabolism, circadian rhythm, DNA repair, and apoptosis. The key cold-related miRNAs, TFs, and genes and their potential regulatory relationships identified in this study provide a deeper understanding of the complex molecular basis of the response to low-temperature environments in the grouper. In particular, our results provide the first identification for the role of NR1D2 gene in the cold tolerance of fish via the regulation of circadian rhythm. Furthermore, the key miRNAs and genes provide a basis for the molecular breeding of new cold-tolerant varieties of the pearl gentian grouper.
</div>

## 
#### Classification of Small RNAs and Identification of miRNAs
<div class="card">
The abundances of known miRNAs, novel miRNAs, rRNAs, tRNAs, snRNAs, snoRNAs, and other sequences in each sample are listed in Table 3. Among small RNA reads, the most frequent type was known miRNAs, followed by rRNAs, snoRNAs, snRNAs, and tRNAs.
</div>

<p style="text-align:center">Table 3 Small RNA (sRNA) classification and annotation statistics</p>

| Sample   | Total Reads | Known miRNA | rRNA   | tRNA | snRNA | snoRNA | Novel miRNA | Other   |
|----------|-------------|-------------|--------|------|-------|--------|-------------|---------|
| CT-1     | 14920658    | 11304891    | 214784 | 1    | 3869  | 15853  | 4749        | 3376511 |
| CT-2     | 14530233    | 10505240    | 161608 | 1    | 4663  | 14042  | 4540        | 3840139 |
| CT-3     | 14864478    | 12006362    | 163918 | 5    | 4716  | 11873  | 4765        | 2672839 |
| LT20-1   | 13083951    | 10945557    | 231819 | 2    | 4898  | 13110  | 3748        | 1884817 |
| LT20-2   | 15418285    | 13190221    | 170333 | 1    | 4873  | 11749  | 4638        | 2036470 |
| LT20-3   | 14379732    | 12200130    | 170457 | 0    | 3283  | 12822  | 4016        | 1989024 |
| LT15-1   | 15172884    | 13018846    | 166328 | 2    | 2657  | 12266  | 3807        | 1968978 |
| LT15-2   | 13632223    | 11791519    | 164365 | 4    | 3553  | 11434  | 3836        | 1657512 |
| LT15-3   | 13060109    | 11324110    | 207754 | 8    | 4198  | 14298  | 4464        | 1505277 |
| LT12-1   | 15850188    | 13397755    | 205844 | 1    | 3384  | 14339  | 6740        | 2222125 |
| LT12-2   | 16095308    | 13317671    | 195947 | 4    | 3867  | 15872  | 4601        | 2557346 |
| LT12-3   | 13103369    | 10978179    | 134694 | 1    | 4133  | 10612  | 5220        | 1970530 |
| LT12-6-1 | 14230098    | 12211442    | 124718 | 1    | 2055  | 10413  | 4168        | 1877301 |
| LT12-6-2 | 14894306    | 12037575    | 140599 | 1    | 3257  | 10949  | 3711        | 2698214 |
| LT12-6-3 | 13709703    | 11892933    | 129790 | 1    | 2869  | 8418   | 3899        | 1671793 |

## 
#### Identification of Novel miRNAs
<div class="card">
In total, 668 known miRNAs were identified by mapping small RNA tags to the miRbase database. These miRNAs belonged to various miRNA families, such as mir0124, mir0190, mir0205, mir0218, and mir0455. In an analysis of the first base of mature miRNAs, adenine (A) or uracil (U) was the primary first bases. In addition, 48 novel miRNAs were predicted, of which 12 novel miRNAs were expressed in all samples, including novel_2, novel_7, novel_10, novel_33, novel_34, novel_38, novel_44, novel_47, novel_63, novel_69, novel_77, and novel_85. The secondary structures of these novel miRNAs were drawn to show the detailed stem-loop structure and mature miRNA sequences (indicated by red lines in Figure 2).
</div>
<br/>
<img src="https://benben-miao.gitee.io/image-cloud/BioBlog/paper2-figure3.png">

Figure. 3 Novel miRNAs and stem-loop structures identified in all samples.

## 
#### Differential Expression Analysis of Known miRNAs and Novel miRNA
<div class="card">
We detected 27 (5 up- and 22 down-regulated), 23 (5 up- and 18 down-regulated), 15 (1 up- and 14 down-regulated), and 38 (13 up- and 25 down-regulated) DE miRNAs in the CT vs. LT20, CT vs. LT15, CT vs. LT12, and CT vs. LT12-6 comparisons, respectively (Fig. 4, Table S2-S5). Notably, 4 DE miRNAs were shared among the four comparisons. Moreover, 4 (1 up- and 3 down-regulated), 11 (4 up- and 7 down-regulated), 28 (12 up- and 16 down-regulated), 5 (2 up- and 3 down-regulated), 24 (12 up- and 12 down-regulated), and 17 (6 up- and 11 down-regulated) DE miRNAs were observed in the LT20 vs. LT15, LT20 vs. LT12, LT20 vs. LT12-6, LT15 vs. LT12, LT15 vs. LT12-6, and LT12 vs. LT12-6 comparisons, respectively. Obviously, the most DE miRNAs (38) were screened in the comparison between the CT group and the LT12-6 group. In addition, 89 unique miRNAs were screened in the pairwise comparisons among all groups, and 72 unique miRNAs were detected in the pairwise comparisons between the control group and the four low-temperature groups.
</div>
<br/>
<img src="https://benben-miao.gitee.io/image-cloud/BioBlog/paper2-figure4.png">

Figure. 4 Volcano plot showing the up-regulation or down-regulation of DE miRNAs in pairwise comparisons between the control group and four low-temperature groups. Dark gray dots (NS): miRNAs without significant differences between groups. Red dots (P & Log2FC): miRNAs that meet the thresholds of P < 0.05 and log2FC > 1.

## 
#### GO and KEGG Enrichment Analyses of DE miRNAs Target Genes
<div class="card">
GO and KEGG pathway enrichment analyses of the DE miRNAs in the pairwise comparisons among groups were performed to further evaluate the functions of target genes regulated by miRNAs under cold stress. The main GO enrichment terms in all pairwise comparisons were generally similar (Fig. 5A, C), including metabolic process, single-organism process, and cellular process in the biological process category, cell, cell part, and membrane in the cellular component category, and binding, catalytic activity, and transporter activity in the molecular function category. Several common pathways, such as vitamin digestion and absorption, biotin metabolism, AMPK signaling pathway, fat digestion and absorption, and phagosome pathways were within the top 20 significantly enriched pathways in the KEGG enrichment analysis (Fig. 5B, D).
</div>
<br/>
<img src="https://benben-miao.gitee.io/image-cloud/BioBlog/paper2-figure5.png">

Figure. 5 Summary of GO and KEGG pathway enrichment analyses of DE miRNAs in the CT vs. LT15 (A-B) and CT vs. LT12-6 (C-D) comparisons.

## 
#### Expression Profiles of miRNA with Temperatures
<div class="card">
Using STEM trend analysis, 20 expression profiles were obtained for all miRNAs in the five groups, and expression changes in profile 0, profile 2, and profile 18 were statistically significant. As illustrated in Fig. 6A, profile 0 was characterized by the overall down-regulation of miRNAs and profile 18 involved an overall up-regulation of miRNAs. The expression trend in profile 2 was unclear. In particular, 79 consistently down-regulated miRNAs were assigned to profile 0, most of which showed slight expression changes (Fig. 6B). In profile 0, the normalized range of fold change values [log2(V(i)/V(0)] for most miRNAs with down-regulated expression was 0 to -4 [|Fold Change|∈(1, 16)], and a few miRNAs had values from -4 to -8 [|Fold Change|∈ (16, 256)]. In contrast, 32 miRNAs were consistently up-regulated in profile 18 (Fig. 6C), most of which were up-regulated in the LT20 group. Expression trends in profile 18 varied. Some miRNAs maintained their original expression levels after an initial down- or up-regulation, while others were continuously down- or up-regulated at all temperatures. The normalized range of values for up-regulated expression for the 32 miRNAs was 0 to 8 [|Fold Change|∈(1, 256)].
</div>
<br/>
<img src="https://benben-miao.gitee.io/image-cloud/BioBlog/paper2-figure6.png">

Figure. 6 STEM analysis and corresponding miRNA expression profiles. (A) Expression profiles of consistently up- or down-regulated miRNAs were screened (P < 0.05). (B) Profile 0: miRNAs were uniformly down-regulated in all groups. (C) Profile 18: miRNAs were uniformly up-regulated in all groups.

## 
#### Identification and Differential Expression Analysis of TFs
<div class="card">
In the present study, 917 TFs were predicted against the Animal TFDB, and these could be assigned to 46 TF families. The top three TF families with the largest number of TFs were TF_bZIP (131 TFs), zf-C2H2 (124 TFs), and bHLH (103 TFs). In total, 63 (40 up- and 23 down-regulated), 138 (84 up- and 54 down-regulated), 148 (110 up- and 38 down-regulated), and 266 (143 up- and 123 down-regulated) TFs were differentially expressed in CT vs. LT20, CT vs. LT15, CT vs. LT12, and CT vs. LT12-6, respectively (Fig. 8, Table S6-S9). Moreover, 25 (15 up- and 10 down-regulated), 91 (70 up- and 21 down-regulated), 186 (111 up- and 75 down-regulated), 56 (35 up- and 21 down-regulated), 129 (72 up- and 57 down-regulated), and 90 (35 up- and 55 down-regulated) DE TFs were identified in LT20 vs. LT15, LT20 vs. LT12, LT20 vs. LT12-6, LT15 vs. LT12, LT15 vs. LT12-6, and LT12 vs. LT12-6, respectively. The most DE TFs (270) were screened between the CT group and the LT12-6 group. Further analyses revealed that 368 unique DE TFs were obtained in pairwise comparisons among all groups, most of which (328) were obtained in pairwise comparisons between only the control group and the four low-temperature groups.
</div>
<br/>
<img src="https://benben-miao.gitee.io/image-cloud/BioBlog/paper2-figure8.png">

Figure. 8 Number and intersection of DE miRNAs in pairwise comparisons between the control group and four low-temperature groups.

## 
#### Regulatory Network of miRNA-TF-mRNA
<div class="card">
The direct regulation of target genes by miRNAs and indirect regulation of genes via TFs were both observed in the miRNA-TF-mRNA regulatory network (Fig. 9). For example, gmo-miR-221-5p directly regulated CYP2J2 (Cytochrome P450 2J2), CYP4B1 (Cytochrome P450 4B1), TM7SF3 (Transmembrane 7 Superfamily Member 3), and GYS2 (Glycogen Synthase 2), ssa-miR-25-3-5p regulated TSKU (Tsukushi, Small Leucine Rich Proteoglycan) and NR1D2 (Nuclear Receptor Subfamily 1 Group D Member 2), and ccr-miR-489 and ccr-miR-122 regulated PPP1CC-A (Protein Phosphatase 1 Catalytic Subunit Gamma) and FOXK2 (Forkhead Box K2), respectively. The miRNA ssa-miR-7132b-5p indirectly regulated EHHADH (Enoyl-CoA Hydratase And 3-Hydroxyacyl CoA Dehydrogenase), ACAT2 (Acetyl-CoA Acetyltransferase 2), HADHA (Hydroxyacyl-CoA Dehydrogenase Trifunctional Multienzyme Complex Subunit Alpha), PCK1, PCK2 (Phosphoenolpyruvate Carboxykinase 1, 2) via the TF ACSS2 (Acyl-CoA Synthetase Short Chain Family Member 2), ola-let-7c indirectly regulated RXRAA, RXRBA, RXRGB (Retinoid X Receptor Alpha, Beta, Gamma) by the regulation of the TF PPARD (Peroxisome Proliferator Activated Receptor Delta), and gmo-miR-10545-5p had an indirect effect on PPP2R1A (Protein Phosphatase 2 Regulatory Subunit 1) and PTPA (Protein Phosphatase 2 Phosphatase Activator) by regulating the TF PPP4CB (Protein Phosphatase 4 Catalytic Subunit). In brief, these regulatory mechanisms and relationships may contribute to the response to cold stress in the pearl gentian grouper.
</div>
<br/>
<img src="https://benben-miao.gitee.io/image-cloud/BioBlog/paper2-figure9.png">

Figure. 9 Potential regulatory network based on the DE miRNAs, DE TFs, and DEGs.

## 
#### Discussion
<div class="card">
Based on an integrated analysis of small RNA sequencing data and previous RNA-Seq data, we obtained 89 DE miRNAs, 368 DE TFs, and 3271 DEGs involved in the response to cold stress in the pearl gentian grouper. A miRNA-TF-mRNA network was constructed based on 23 DE miRNAs, 24 DE TFs, and 91 DEGs. By further exploring the functions and pathways related to these molecules, we finally confirmed that 28 key molecules, including 7 DE miRNAs, 3 DE TFs, and 18 DEGs, were involved in several important pathways and biological reactions associated with cold stress in pearl gentian grouper by both direct and indirect regulatory relationships (Fig. 11). The main functions identified in this analysis could be summarized into four general categories (Fig. 11): antioxidation and membrane fluidity, glucose and lipid metabolisms, circadian rhythm, and DNA repair and apoptosis.
</div>
<br/>
<img src="https://benben-miao.gitee.io/image-cloud/BioBlog/paper2-figure11.png">

Figure. 11 Several important pathways and biological reactions associated with the miRNA-TF-mRNA regulatory network involved in the response to cold stress in pearl gentian grouper.

## 
#### Conclusion
<div class="card">
In this study, DE miRNAs, mRNAs, and TFs in pearl gentian grouper were screened, and a candidate miRNA-TF-mRNA regulatory network related to cold stress was constructed. The network involved two regulatory mechanisms, i.e., four miRNAs directly regulated eight target genes and three miRNAs indirectly regulated ten target genes via the regulation of three TFs. These miRNAs, TFs, and mRNAs played essential roles in several important pathways and biological reactions in pearl gentian grouper under cold stress, such as antioxidant and membrane fluidity, glucose and lipid metabolism, circadian rhythm regulation, DNA repair, and autophagy. In short, the two regulatory mechanisms revealed in pearl gentian grouper protected against damage caused by cold stress and promoted the maintenance of normal physiological activity. Of note, the potential function of NR1D2 gene in cold tolerance in fish via the regulation of the circadian rhythm was identified for the first time in this study. The complex regulatory relationships revealed in the present study not only provide a deeper understanding of the molecular mechanism underlying the adaptation of pearl gentian grouper to low-temperature environments but also provide clear directions for cultivating cold-tolerant varieties in the future.
</div>

## 
#### References
<div class="card">
Agarwal, V., Bell, G.W., Nam, J.W., Bartel, D.P., 2015. Predicting effective microRNA target sites in mammalian mRNAs. Elife 4. https://doi.org/10.7554/eLife.05005 

Amaral, I.P.G., Johnston, I.A., 2012. Circadian expression of clock and putative clock-controlled genes in skeletal muscle of the zebrafish. Am. J. Physiol. - Regul. Integr. Comp. Physiol. 302. https://doi.org/10.1152/ajpregu.00367.2011

Bartel, D.P., 2009. MicroRNAs: Target Recognition and Regulatory Functions. Cell. https://doi.org/10.1016/j.cell.2009.01.002

...
</div>